db/models.py

```diff
@@ -46,3 +50,73 @@ class MovieSession(models.Model):
 
     def __str__(self) -> str:
         return f"{self.movie.title} {str(self.show_time)}"
+
+
+class User(AbstractUser):
+    pass
+
+
+class Order(models.Model):
+    created_at = models.DateTimeField(auto_now_add=True)
+    user = models.ForeignKey(
+        settings.AUTH_USER_MODEL,
+        on_delete=models.CASCADE
+    )
+
+    class Meta:
+        ordering = ["-created_at"]
+
+    def __str__(self) -> str:
+        return str(self.created_at)
+
+
+class Ticket(models.Model):
+    movie_session = models.ForeignKey(
+        to=MovieSession,
+        on_delete=models.CASCADE
+    )
+    order = models.ForeignKey(to=Order, on_delete=models.CASCADE)
+    row = models.IntegerField()
+    seat = models.IntegerField()
+
+    def __str__(self) -> str:
+        return f"{self.movie_session.movie.title}" \
+               f" {self.movie_session.show_time} " \
+               f"(row: {self.row}, seat: {self.seat})"
+
+    def clean(self) -> None:
+        rows = self.movie_session.cinema_hall.rows
+        if not (1 <= self.row <= rows):
+            raise ValidationError(
+                {"row": [f"row number must be in "
+                         f"available range: (1, rows): (1, {rows})"]}
+            )
+        seats_in_row = self.movie_session.cinema_hall.seats_in_row
+        if not (1 <= self.seat <= seats_in_row):
+            raise ValidationError({
+                "seat": [f"seat number must be in available range: "
+                         f"(1, seats_in_row): (1, {seats_in_row})"]
+            })
+
+    def save(
+            self,
+            force_insert: str = False,
+            force_update: str = False,
+            using: str = False,
+            update_fields: str = None
```

Comment:
Use Optional[<value type>] for annotations. Fix everythere where the default value is None